src/plugins/auto/auto_schedule.cpp

```diff
@@ -301,6 +406,23 @@ void AutoSchedule::init(const ScheduleContext::Ptr& sContext) {
             }
         };
         _executor->run(std::move(recycleTask));
+    } else if (_autoSContext->_devicePriorities.size() != 1 && !isCumulative && _autoSContext->_runtimeFallback) {
+        _firstLoadFuture = _firstLoadPromise.get_future();
+        // will not wait for loading accelerator network,
+        // so the executor can't be destroyed before finished the task,
+        // so use executor as a member of AutoSchedule.
+        _executor = _autoSContext->_plugin->executorManager()->getIdleCPUStreamsExecutor(
+                    IE::IStreamsExecutor::Config{"AutoDeviceAsyncLoad",
+                    static_cast<int>(std::thread::hardware_concurrency()) /* max possible #streams*/,
+                    0 /*default threads per stream, workaround for ticket 62376*/,
+                    IE::IStreamsExecutor::ThreadBindingType::NONE});
+        for (auto&& device : _autoSContext->_devicePriorities) {
+            // initialize containers before run async task
+            _idleWorkerRequests[device.deviceName];
+            _workerRequests[device.deviceName];
+            _inferPipelineTasksDeviceSpecific[device.deviceName] = nullptr;
+        }
+        _executor->run(_loadContext[ACTUALDEVICE].task);
```

Comment:
the _passthroughExeNet is enabled for single device from memory copy saving perspective (to pass through infer requests directly to hardware infer requests), we shall not break this rule. with this implementation, extra memcpy will be introduced.